src/groupByKey.ts

```diff
@@ -2,6 +2,17 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey<T>(items: T[], key: keyof T): GroupsMap<T> {
+  const newObj: GroupsMap<T> = {};
+
+  items.forEach((el: T) => {
```

Comment:
```suggestion
  items.forEach((item: T) => {
```

I'd better call it `item` 🙂